dev/run

```diff
@@ -524,6 +544,20 @@ def hack_local_ini(ctx, contents):
     return contents + "\n\n[chttpd_auth]\nsecret = %s\n" % COMMON_SALT
 
 
+def hack_vm_args(ctx, node, contents):
+    contents += f"""
+-proto_dist couch
+-couch_dist no_tls '"clouseau@127.0.0.1"'
```

Comment:
each node will talk to a different clouseau (clouseau1 for node1, clouseau2 for node2, etc)

Reply:
Changed, thanks for the review!